docs/runtime-contract.md

```diff
@@ -381,10 +350,10 @@ such variables will follow demonstrated usage and utility.
 ### User
 
 Developers MAY specify that containers should be run as a specific user or group
-ID using the `runAsUser` container property. If specified, the runtime MUST run
+ID using the `runAsUser` container property. If specified, the runtime
+[MUST](https://github.com/knative/serving/blob/master/test/conformance/user_test.go) run
 the container as the specified user ID if allowed by the platform (see below).
-If no `runAsUser` is specified, a platform-specific default SHALL be used.
-Platform Providers SHOULD document this default behavior.
```

Comment:
This is removing a documentation recommendation.

Would it make sense instead to have a section of the spec recommending the most significant documentation areas where platform providers may differ?

Reply:
I am going to take another stab at updating this after some more recent conversations about how OpenShift handles `runAsUser`. 

It would be nice to have documentation recommendations and requirements elsewhere so you don't have to go through this entire document to find them all.